lib/spack/spack/installer.py

```diff
@@ -280,6 +280,32 @@ def _print_installed_pkg(message):
     print(colorize("@*g{[+]} ") + spack.util.path.debug_padded_filter(message))
 
 
+def _print_install_test_log(pkg, verbose):
+    """Output install test log file information.
+
+    Args:
+        pkg (spack.package.PackageBase): package of interest
+        verbose (bool): True if the test log contents are to be printed
+    """
+    if not pkg.run_tests:
+        # The tests were not run
+        return
+
+    log = pkg.install_test_install_log_path
+    if not os.path.isfile(log):
+        log = pkg.test_install_log if hasattr(pkg, "test_install_log") else None
+        if not (log and os.path.isfile(log)):
+            tty.debug("There is no test log file (staged or installed)")
+            return
+
+    if verbose:
+        with open(log, "r") as f:
+            for ln in f.readlines():
+                print(ln.strip("\n"))
+
+    print("\nSee test results at:\n  {0}".format(log))
```

Comment:
I don't know if we need to print this unless the tests fail. We don't print spack-build-out.txt and friends unless the install fails.

Reply:
Fair enough.

Reply:
My main reservation here is that it's handy to be reminded where the test outputs are in case you want to make sure all of the tests you expect to be executed (e.g., inherited, including from virtual) are actually run. 

I have also seen "passing" tests that encounter failures that aren't trapped/tracked by the test implementation.   I should make a list of these when I see them.

Reply:
Likewise, it's handy to be reminded where `spack-build-out.txt` is, but I don't think we need to do that by default on every package every time. Users can use `--verbose` if they want to see which tests were executed.

Reply:
True.  (I already changed this BTW.)